definitions/npm/ramda_v0.26.x/flow_v0.104.x-/ramda_v0.26.x.js

```diff
@@ -777,11 +777,14 @@ declare module ramda {
     fn: UnaryPredicateFn<T>,
   ): (xs: Array<T>) => boolean;
 
-  declare function nth<V, T: Array<V>>(i: number, xs: T): ?V;
-  declare function nth<V, T: Array<V> | string>(
-    i: number,
-  ): ((xs: string) => string) & ((xs: T) => ?V);
-  declare function nth<T: string>(i: number, xs: T): T;
+  declare var nth: {|
+    <A, As: $ReadOnlyArray<A>>(n: number, xs: As): ?A,
+    <A, As: $ReadOnlyArray<A> | string>(
+      n: number,
+    ): ((xs: string) => string) & ((xs: As) => ?A),
+    (n: number, xs: string): string,
+  |}
```

Comment:
I think you've mixed your `nth` and `sortBy` pull requests together. It's probably small enough that we can do them in the same pull request, but if we're to do that then let's update the pull request title and description.

Reply:
Not intentionally, was just trying to save some time resolving the merge conflict within web console.